backend/app/services/openapi/chat_response.py

```
@@ -140,6 +140,54 @@ async def _create_streaming_response_http(
+                            "name": name,
+                            "url": config.get("url", ""),
+                            "type": config.get("type", "streamable-http"),
+                        }
+                        if config.get("headers"):
+                            server_entry["auth"] = config["headers"]
+                        mcp_servers_to_pass.append(server_entry)
+                logger.info(
+                    f"[OPENAPI_HTTP] Loaded {len(bot_mcp_servers)} bot MCP servers for HTTP mode: "
+                    f"bot={setup.bot_namespace}/{setup.bot_name}"
+                )
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)